pylintrc

```diff
@@ -14,6 +14,7 @@ max-line-length=131
 
 [TYPECHECK]
 ignored-classes=pytest,lxml.etree
+ignored-modules=distutils
```

Comment:
We could drop the `if True:` junk and just rely on this line if we upgraded pylint ([some familiar faces here](https://bitbucket.org/logilab/pylint/issues/223/ignored-modules-should-turn-no-name-in)) but then we'd lose support for Python 2.6.